### DISCUSSION TITLE
SOLVED Digital Ocean droplet - confirmation e-mail is not delivered

### DISCUSSION THREAD

**Discussion by olivievranska** [mastodon/mastodon#19675] [General]
Hello,

I'm a newbie in mastodon and a bit experienced Unix admin. 

I want to run my own Mastodon instance, so I bought a domain cztwitter.cz, used 1-click Digital Ocean droplet and Mastodon website is running.

But I haven't got confirmation e-mail for the first user.

.env Production SMTP settings:
SMTP_SERVER=smtp.cztwitter.cz
SMTP_PORT=465
SMTP_LOGIN=mastodon@cztwitter.cz
SMTP_PASSWORD=<PASSWORD>
SMTP_FROM_ADDRESS=mastodon@cztwitter.cz
SMTP_AUTH_METHOD=plain
SMTP_OPENSSL_VERIFY_MODE=none

I tried to send test e-mail by this account to the same e-mail address as registration is from e-mail client with SMTP settings below and e-mail was delivered:
SMTP Server: smtp.cztwitter.cz
Port: 465
Encrypting: SSL/TLS
Requires authentification: yes

Please, can you suggest how to solve this issue?

edit:
Solved. Webhosting of domain don't have SMTP API, SMTP setting replaced by mailgun.

**Comment by ralekna** [comment]
Hi, could you explain how SMTP server doesn't have an API? Is it some standard? Or it's it doesn't have SMTP server at all?

**Reply by DavidGerhart** [reply]
I think what the OP is saying about the solution for their instance is this: That there are DNS records that must exist for those Mastodon settings to function. For instance: My Domain is registered at a provider, where I reference nameservers at DigitalOcean. Within the DNS management tools there, I installed the MX records, A records, Cname ... for the SMTP Relay service I'm using. 

Admittedly, I still have problems, but I'm getting closer...  ;-)

**Comment by jakedfw** [comment]
A great option here is mailersend or sendinblue, both with free SMTP relay services.
